.github/workflows/push.yml

```diff
@@ -0,0 +1,22 @@
+name: maven-build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
```

Comment:
Shouldn't this be java 17?

Reply:
Actually, wasn't MC 1.16 still java 8?

Reply:
1.16 supports 8 yeah, but 1.17 only supports java 16 and 1.18 only java 17.

Reply:
but backwards compatibility exists :)
I think I will bump to JDK 17, since that is probably what we use to compile as well, but I think maven still compiles with java 8 support.